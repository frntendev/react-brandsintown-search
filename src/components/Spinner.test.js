import React from "react";
import { shallow, mount, configure } from "enzyme";
import Spinner from "./Spinner";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });

describe("Spinner", () => {
  test("renders component itself correctly", () => {
    const renderedComponent = shallow(<Spinner />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
