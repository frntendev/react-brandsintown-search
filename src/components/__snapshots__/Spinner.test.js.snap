// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders component itself correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Spinner />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <svg
        fill="#49c5b6"
        height="30"
        viewBox="0 0 120 30"
        width="120"
        xmlns="http://www.w3.org/2000/svg"
      >
        <circle
          cx="15"
          cy="15"
          r="15"
        >
          <animate
            attributeName="r"
            begin="0s"
            calcMode="linear"
            dur="0.8s"
            from="15"
            repeatCount="indefinite"
            to="15"
            values="15;9;15"
          />
          <animate
            attributeName="fill-opacity"
            begin="0s"
            calcMode="linear"
            dur="0.8s"
            from="1"
            repeatCount="indefinite"
            to="1"
            values="1;.5;1"
          />
        </circle>
        <circle
          cx="60"
          cy="15"
          fillOpacity="0.3"
          r="9"
        >
          <animate
            attributeName="r"
            begin="0s"
            calcMode="linear"
            dur="0.8s"
            from="9"
            repeatCount="indefinite"
            to="9"
            values="9;15;9"
          />
          <animate
            attributeName="fill-opacity"
            begin="0s"
            calcMode="linear"
            dur="0.8s"
            from="0.5"
            repeatCount="indefinite"
            to="0.5"
            values=".5;1;.5"
          />
        </circle>
        <circle
          cx="105"
          cy="15"
          r="15"
        >
          <animate
            attributeName="r"
            begin="0s"
            calcMode="linear"
            dur="0.8s"
            from="15"
            repeatCount="indefinite"
            to="15"
            values="15;9;15"
          />
          <animate
            attributeName="fill-opacity"
            begin="0s"
            calcMode="linear"
            dur="0.8s"
            from="1"
            repeatCount="indefinite"
            to="1"
            values="1;.5;1"
          />
        </circle>
      </svg>,
      "className": "css-182awlh",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <circle
            cx="15"
            cy="15"
            r="15"
          >
            <animate
              attributeName="r"
              begin="0s"
              calcMode="linear"
              dur="0.8s"
              from="15"
              repeatCount="indefinite"
              to="15"
              values="15;9;15"
            />
            <animate
              attributeName="fill-opacity"
              begin="0s"
              calcMode="linear"
              dur="0.8s"
              from="1"
              repeatCount="indefinite"
              to="1"
              values="1;.5;1"
            />
          </circle>,
          <circle
            cx="60"
            cy="15"
            fillOpacity="0.3"
            r="9"
          >
            <animate
              attributeName="r"
              begin="0s"
              calcMode="linear"
              dur="0.8s"
              from="9"
              repeatCount="indefinite"
              to="9"
              values="9;15;9"
            />
            <animate
              attributeName="fill-opacity"
              begin="0s"
              calcMode="linear"
              dur="0.8s"
              from="0.5"
              repeatCount="indefinite"
              to="0.5"
              values=".5;1;.5"
            />
          </circle>,
          <circle
            cx="105"
            cy="15"
            r="15"
          >
            <animate
              attributeName="r"
              begin="0s"
              calcMode="linear"
              dur="0.8s"
              from="15"
              repeatCount="indefinite"
              to="15"
              values="15;9;15"
            />
            <animate
              attributeName="fill-opacity"
              begin="0s"
              calcMode="linear"
              dur="0.8s"
              from="1"
              repeatCount="indefinite"
              to="1"
              values="1;.5;1"
            />
          </circle>,
        ],
        "fill": "#49c5b6",
        "height": "30",
        "viewBox": "0 0 120 30",
        "width": "120",
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <animate
                attributeName="r"
                begin="0s"
                calcMode="linear"
                dur="0.8s"
                from="15"
                repeatCount="indefinite"
                to="15"
                values="15;9;15"
              />,
              <animate
                attributeName="fill-opacity"
                begin="0s"
                calcMode="linear"
                dur="0.8s"
                from="1"
                repeatCount="indefinite"
                to="1"
                values="1;.5;1"
              />,
            ],
            "cx": "15",
            "cy": "15",
            "r": "15",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "attributeName": "r",
                "begin": "0s",
                "calcMode": "linear",
                "dur": "0.8s",
                "from": "15",
                "repeatCount": "indefinite",
                "to": "15",
                "values": "15;9;15",
              },
              "ref": null,
              "rendered": null,
              "type": "animate",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "attributeName": "fill-opacity",
                "begin": "0s",
                "calcMode": "linear",
                "dur": "0.8s",
                "from": "1",
                "repeatCount": "indefinite",
                "to": "1",
                "values": "1;.5;1",
              },
              "ref": null,
              "rendered": null,
              "type": "animate",
            },
          ],
          "type": "circle",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <animate
                attributeName="r"
                begin="0s"
                calcMode="linear"
                dur="0.8s"
                from="9"
                repeatCount="indefinite"
                to="9"
                values="9;15;9"
              />,
              <animate
                attributeName="fill-opacity"
                begin="0s"
                calcMode="linear"
                dur="0.8s"
                from="0.5"
                repeatCount="indefinite"
                to="0.5"
                values=".5;1;.5"
              />,
            ],
            "cx": "60",
            "cy": "15",
            "fillOpacity": "0.3",
            "r": "9",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "attributeName": "r",
                "begin": "0s",
                "calcMode": "linear",
                "dur": "0.8s",
                "from": "9",
                "repeatCount": "indefinite",
                "to": "9",
                "values": "9;15;9",
              },
              "ref": null,
              "rendered": null,
              "type": "animate",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "attributeName": "fill-opacity",
                "begin": "0s",
                "calcMode": "linear",
                "dur": "0.8s",
                "from": "0.5",
                "repeatCount": "indefinite",
                "to": "0.5",
                "values": ".5;1;.5",
              },
              "ref": null,
              "rendered": null,
              "type": "animate",
            },
          ],
          "type": "circle",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <animate
                attributeName="r"
                begin="0s"
                calcMode="linear"
                dur="0.8s"
                from="15"
                repeatCount="indefinite"
                to="15"
                values="15;9;15"
              />,
              <animate
                attributeName="fill-opacity"
                begin="0s"
                calcMode="linear"
                dur="0.8s"
                from="1"
                repeatCount="indefinite"
                to="1"
                values="1;.5;1"
              />,
            ],
            "cx": "105",
            "cy": "15",
            "r": "15",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "attributeName": "r",
                "begin": "0s",
                "calcMode": "linear",
                "dur": "0.8s",
                "from": "15",
                "repeatCount": "indefinite",
                "to": "15",
                "values": "15;9;15",
              },
              "ref": null,
              "rendered": null,
              "type": "animate",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "attributeName": "fill-opacity",
                "begin": "0s",
                "calcMode": "linear",
                "dur": "0.8s",
                "from": "1",
                "repeatCount": "indefinite",
                "to": "1",
                "values": "1;.5;1",
              },
              "ref": null,
              "rendered": null,
              "type": "animate",
            },
          ],
          "type": "circle",
        },
      ],
      "type": "svg",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <svg
          fill="#49c5b6"
          height="30"
          viewBox="0 0 120 30"
          width="120"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="15"
            cy="15"
            r="15"
          >
            <animate
              attributeName="r"
              begin="0s"
              calcMode="linear"
              dur="0.8s"
              from="15"
              repeatCount="indefinite"
              to="15"
              values="15;9;15"
            />
            <animate
              attributeName="fill-opacity"
              begin="0s"
              calcMode="linear"
              dur="0.8s"
              from="1"
              repeatCount="indefinite"
              to="1"
              values="1;.5;1"
            />
          </circle>
          <circle
            cx="60"
            cy="15"
            fillOpacity="0.3"
            r="9"
          >
            <animate
              attributeName="r"
              begin="0s"
              calcMode="linear"
              dur="0.8s"
              from="9"
              repeatCount="indefinite"
              to="9"
              values="9;15;9"
            />
            <animate
              attributeName="fill-opacity"
              begin="0s"
              calcMode="linear"
              dur="0.8s"
              from="0.5"
              repeatCount="indefinite"
              to="0.5"
              values=".5;1;.5"
            />
          </circle>
          <circle
            cx="105"
            cy="15"
            r="15"
          >
            <animate
              attributeName="r"
              begin="0s"
              calcMode="linear"
              dur="0.8s"
              from="15"
              repeatCount="indefinite"
              to="15"
              values="15;9;15"
            />
            <animate
              attributeName="fill-opacity"
              begin="0s"
              calcMode="linear"
              dur="0.8s"
              from="1"
              repeatCount="indefinite"
              to="1"
              values="1;.5;1"
            />
          </circle>
        </svg>,
        "className": "css-182awlh",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <circle
              cx="15"
              cy="15"
              r="15"
            >
              <animate
                attributeName="r"
                begin="0s"
                calcMode="linear"
                dur="0.8s"
                from="15"
                repeatCount="indefinite"
                to="15"
                values="15;9;15"
              />
              <animate
                attributeName="fill-opacity"
                begin="0s"
                calcMode="linear"
                dur="0.8s"
                from="1"
                repeatCount="indefinite"
                to="1"
                values="1;.5;1"
              />
            </circle>,
            <circle
              cx="60"
              cy="15"
              fillOpacity="0.3"
              r="9"
            >
              <animate
                attributeName="r"
                begin="0s"
                calcMode="linear"
                dur="0.8s"
                from="9"
                repeatCount="indefinite"
                to="9"
                values="9;15;9"
              />
              <animate
                attributeName="fill-opacity"
                begin="0s"
                calcMode="linear"
                dur="0.8s"
                from="0.5"
                repeatCount="indefinite"
                to="0.5"
                values=".5;1;.5"
              />
            </circle>,
            <circle
              cx="105"
              cy="15"
              r="15"
            >
              <animate
                attributeName="r"
                begin="0s"
                calcMode="linear"
                dur="0.8s"
                from="15"
                repeatCount="indefinite"
                to="15"
                values="15;9;15"
              />
              <animate
                attributeName="fill-opacity"
                begin="0s"
                calcMode="linear"
                dur="0.8s"
                from="1"
                repeatCount="indefinite"
                to="1"
                values="1;.5;1"
              />
            </circle>,
          ],
          "fill": "#49c5b6",
          "height": "30",
          "viewBox": "0 0 120 30",
          "width": "120",
          "xmlns": "http://www.w3.org/2000/svg",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <animate
                  attributeName="r"
                  begin="0s"
                  calcMode="linear"
                  dur="0.8s"
                  from="15"
                  repeatCount="indefinite"
                  to="15"
                  values="15;9;15"
                />,
                <animate
                  attributeName="fill-opacity"
                  begin="0s"
                  calcMode="linear"
                  dur="0.8s"
                  from="1"
                  repeatCount="indefinite"
                  to="1"
                  values="1;.5;1"
                />,
              ],
              "cx": "15",
              "cy": "15",
              "r": "15",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "attributeName": "r",
                  "begin": "0s",
                  "calcMode": "linear",
                  "dur": "0.8s",
                  "from": "15",
                  "repeatCount": "indefinite",
                  "to": "15",
                  "values": "15;9;15",
                },
                "ref": null,
                "rendered": null,
                "type": "animate",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "attributeName": "fill-opacity",
                  "begin": "0s",
                  "calcMode": "linear",
                  "dur": "0.8s",
                  "from": "1",
                  "repeatCount": "indefinite",
                  "to": "1",
                  "values": "1;.5;1",
                },
                "ref": null,
                "rendered": null,
                "type": "animate",
              },
            ],
            "type": "circle",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <animate
                  attributeName="r"
                  begin="0s"
                  calcMode="linear"
                  dur="0.8s"
                  from="9"
                  repeatCount="indefinite"
                  to="9"
                  values="9;15;9"
                />,
                <animate
                  attributeName="fill-opacity"
                  begin="0s"
                  calcMode="linear"
                  dur="0.8s"
                  from="0.5"
                  repeatCount="indefinite"
                  to="0.5"
                  values=".5;1;.5"
                />,
              ],
              "cx": "60",
              "cy": "15",
              "fillOpacity": "0.3",
              "r": "9",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "attributeName": "r",
                  "begin": "0s",
                  "calcMode": "linear",
                  "dur": "0.8s",
                  "from": "9",
                  "repeatCount": "indefinite",
                  "to": "9",
                  "values": "9;15;9",
                },
                "ref": null,
                "rendered": null,
                "type": "animate",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "attributeName": "fill-opacity",
                  "begin": "0s",
                  "calcMode": "linear",
                  "dur": "0.8s",
                  "from": "0.5",
                  "repeatCount": "indefinite",
                  "to": "0.5",
                  "values": ".5;1;.5",
                },
                "ref": null,
                "rendered": null,
                "type": "animate",
              },
            ],
            "type": "circle",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <animate
                  attributeName="r"
                  begin="0s"
                  calcMode="linear"
                  dur="0.8s"
                  from="15"
                  repeatCount="indefinite"
                  to="15"
                  values="15;9;15"
                />,
                <animate
                  attributeName="fill-opacity"
                  begin="0s"
                  calcMode="linear"
                  dur="0.8s"
                  from="1"
                  repeatCount="indefinite"
                  to="1"
                  values="1;.5;1"
                />,
              ],
              "cx": "105",
              "cy": "15",
              "r": "15",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "attributeName": "r",
                  "begin": "0s",
                  "calcMode": "linear",
                  "dur": "0.8s",
                  "from": "15",
                  "repeatCount": "indefinite",
                  "to": "15",
                  "values": "15;9;15",
                },
                "ref": null,
                "rendered": null,
                "type": "animate",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "attributeName": "fill-opacity",
                  "begin": "0s",
                  "calcMode": "linear",
                  "dur": "0.8s",
                  "from": "1",
                  "repeatCount": "indefinite",
                  "to": "1",
                  "values": "1;.5;1",
                },
                "ref": null,
                "rendered": null,
                "type": "animate",
              },
            ],
            "type": "circle",
          },
        ],
        "type": "svg",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
